stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/DefaultChannelEventsHandler.kt

```diff
@@ -0,0 +1,62 @@
+package io.getstream.chat.android.offline.querychannels
+
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.api.models.FilterObject
+import io.getstream.chat.android.client.api.models.QueryChannelsRequest
+import io.getstream.chat.android.client.call.await
+import io.getstream.chat.android.client.events.ChannelUpdatedByUserEvent
+import io.getstream.chat.android.client.events.ChannelUpdatedEvent
+import io.getstream.chat.android.client.events.HasChannel
+import io.getstream.chat.android.client.events.NotificationAddedToChannelEvent
+import io.getstream.chat.android.client.models.Channel
+import io.getstream.chat.android.client.models.Filters
+import io.getstream.chat.android.client.utils.map
+
+internal class DefaultChannelEventsHandler(
+    private val client: ChatClient,
+    private val filter: FilterObject,
+) : BaseChannelEventsHandler() {
+    internal var checkFilterOnChannelUpdatedEvent: Boolean = false
```

Comment:
I think we can remove it and just not check channels on those events by default

Reply:
But then the behaviour will be changed in a new version. It keeps old logic. I think we should keep it until we delete `QueryChannelsController:: checkFilterOnChannelUpdatedEvent`

Reply:
I would be surprised if anyone is using those flags/newEventFilter at the moment because it wasn't easy to access them but I'm okay with removing it later